app/api/v2/views/users.py

```diff
@@ -1,11 +1,18 @@
-from flask import Flask, jsonify, request
-from flask_jwt_extended import(
-    JWTManager, jwt_optional, create_access_token, get_jwt_identity)
+import re
+from flask import Flask, jsonify, request, Response
+from flask_jwt_extended import(JWTManager, jwt_optional, create_access_token, get_jwt_identity, get_raw_jwt)
+from app.api.v2.models.users import UserModel
+from flask_restful import Resource
+from db import db_connection
+import json
 
 
-class Registration(Resource):
+class Signup(Resource):
     def post(self):
         data = request.get_json(force=True)
```

Comment:
Why do you pass `force=True` in the get_json method?

Reply:
Hello Boniface,
It definitely is not a good practice.
At some point in my code, I had a blocker of not getting the desired outcome and this is the solution presented by a peer and it worked in that occasion.
I seem to have taken it as a norm which should not be the case.

My research says that it has the effect to force Https status and pass custom headers.

I will suspend with it.

Regards
Steven Ennis